docs/content/en/docs-v1.0.x/installation/install-piped/installing-on-kubernetes.md

```diff
@@ -0,0 +1,164 @@
+>
+>We now also have a repository for community built plugins. See the [Community plugins repository on GitHub](https://github.com/pipe-cd/community-plugins)  to know more.
+
+## In the cluster-wide mode
+
+This way requires installing cluster-level resources. Piped installed with this way can perform deployment workloads against any other namespaces than the where Piped runs on.
+
+- Preparing a piped configuration file as the following
+
+  ``` yaml
+  apiVersion: pipecd.dev/v1beta1
+  kind: Piped
+  spec:
+    projectID: {PROJECT_ID}
+    pipedID: {PIPED_ID}
+    pipedKeyFile: /etc/piped-secret/piped-key
+    # Write in a format like "host:443" because the communication is done via gRPC.
+    apiAddress: {CONTROL_PLANE_API_ADDRESS}
+    git:
+      sshKeyFile: /etc/piped-secret/ssh-key
+    repositories:
+      - repoId: {REPO_ID_OR_NAME}
+        remote: git@github.com:{GIT_ORG}/{GIT_REPO}.git
+        branch: {GIT_BRANCH}
+    syncInterval: 1m
+  ```
+
+See [ConfigurationReference](../../../user-guide/managing-piped/configuration-reference/) for the full configuration.
+
+- Installing by using [Helm](https://helm.sh/docs/intro/install/) (3.8.0 or later)
+
+  {{< tabpane >}}
+  {{< tab lang="bash" header="Piped" >}}
+# This command just installs a Piped with the specified version.
+# Whenever you want to upgrade that Piped to a new version or update its config data
+# you have to restart it by re-running this command.
+
+helm upgrade -i dev-piped oci://ghcr.io/pipe-cd/chart/piped --version={{< blocks/latest_version >}} --namespace={NAMESPACE} \
+  --set-file config.data={PATH_TO_PIPED_CONFIG_FILE} \
+  --set-file secret.data.piped-key={PATH_TO_PIPED_KEY_FILE} \
+  --set-file secret.data.ssh-key={PATH_TO_PRIVATE_SSH_KEY_FILE}
+  {{< /tab >}}
+  
+  {{< /tabpane >}}
+
+  Note: Be sure to set `--set args.insecure=true` if your Control Plane has not TLS-enabled yet.
+
+  See [values.yaml](https://github.com/pipe-cd/pipecd/blob/master/manifests/piped/values.yaml) for the full values.
+
+## In the namespaced mode
+
+The previous way requires installing cluster-level resources. If you want to restrict Piped's permission within the namespace where Piped runs on, you can configure it using the scope parameter.
+
+- Installing by using [Helm](https://helm.sh/docs/intro/install/) (3.8.0 or later)
+
+  {{< tabpane >}}
+  {{< tab lang="bash" header="Piped" >}}
+# This command just installs a Piped with the specified version.
+# Whenever you want to upgrade that Piped to a new version or update its config data
+# you have to restart it by re-running this command.
+
+helm upgrade -i dev-piped oci://ghcr.io/pipe-cd/chart/piped --version={{< blocks/latest_version >}} --namespace={NAMESPACE} \
+  --set-file config.data={PATH_TO_PIPED_CONFIG_FILE} \
+  --set-file secret.data.piped-key={PATH_TO_PIPED_KEY_FILE} \
+  --set-file secret.data.ssh-key={PATH_TO_PRIVATE_SSH_KEY_FILE} \
+  --set args.enableDefaultKubernetesCloudProvider=false \
```

Comment:
```suggestion
```
we don't have this flag for pipedv1
ref: https://pipecd.dev/blog/2025/09/02/what-is-new-in-pipedv1-plugin-arch-piped/#weird-flag---enable-default-kubernetes-cloud-provider-in-piped-execution